flash_examples/finetuning/style_transfer.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+import flash
+from flash.data.utils import download_data
+from flash.utils.imports import _PYSTICHE_AVAILABLE
+from flash.vision.style_transfer import StyleTransfer, StyleTransferData
+
+if _PYSTICHE_AVAILABLE:
```

Comment:
If `pystiche` is not available, we'll never get to this, since this PR makes `flash` dependent on `pystiche` at import. Should this not be the case?

Reply:
We will refactor requirement, but Flash will depend on  pystiche only if user do `pip install flash['style-transfer']` and we need to make sure the code doesn't break your library isn't installed.

Reply:
But until this happens, this check is moot. Even after the refactor it is probably useless, since importing `StyleTransfer` depends on `pystiche`, so we need to do the guarding there.

Reply:
After #297 `pystiche` is now an optional dependency.